scipy/spatial/transform/tests/test_rotation.py

```diff
@@ -586,10 +586,20 @@ def test_as_euler_asymmetric_axes():
     for seq_tuple in permutations('xyz'):
         # Extrinsic rotations
         seq = ''.join(seq_tuple)
-        assert_allclose(angles, Rotation.from_euler(seq, angles).as_euler(seq))
+        
+        rotation = Rotation.from_euler(seq, angles)
+        angles_from_quat = rotation.as_euler(seq)
+        angles_from_matrix = rotation.as_euler_from_matrix(seq)
+        assert_allclose(angles, angles_from_quat)
```

Comment:
As you've pointed out it doesn't look that logical to test it like this...

From a rational point of view it's likely better to just replace the method and verify it with tests (which are quite extensive).

Reply:
Alright then! Do you want me to fallback the test_rotation file to as it was before?